pkg/codegen/testing/test/program_driver.go

```diff
@@ -200,6 +204,85 @@ var PulumiPulumiProgramTests = []ProgramTest{
 		Skip:        codegen.NewStringSet("go"),
 		// Blocked on go: TODO[pulumi/pulumi#10834]
 	},
+	// PCL files from pulumi/yaml transpiled examples
```

Comment:
Since we are automating pulling the contents of `transpiled_examples`, we should automate ensuring they are under test here.

At the very least, we can assert that when a test is run, all expected values of `transpiled_examples` are present. That way if a new example is added in pulumi-yaml, and we don't update program_driver, we will get a test failure to remind us.